clean_user_registry_secrets/clean_user_registry_secrets.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Scripts used to remove user registry secrets in k8s"""
+
+from datetime import datetime, timedelta
```

Comment:
@ableuler I assume that you still keep the python builtin packages vs the ones that are installed separate right? So because the only non-built-in package is kubernetes I only had to sort the group of built in packages.